tests/robustness/validate/watch.go

```diff
@@ -125,14 +125,18 @@ func validateBookmarkable(lg *zap.Logger, report report.ClientReport) (err error
 
 func validateOrdered(lg *zap.Logger, report report.ClientReport) (err error) {
```

Comment:
Validate ordered is not related to testing watch being sequential. Ordered is a per watch guarantee defined in https://etcd.io/docs/v3.6/learning/api_guarantees/#watch-apis.

For testing watch being sequential we should have w separate validation that considers only watch with revision equal 0, and checks only their first and last events per member Id.